pecan_swagger/swag/restswag.py

```diff
@@ -75,7 +76,20 @@ def collect_methods(controllers):
             pass
         # pecan's _lookup function
         if getattr(controller[0], '_lookup', None):
-            pass
+            lookup = controller[0]._lookup
+            routes = lookup.__routes
```

Comment:
probably want to move this under the "if" statement to ensure that `__routes` exists